module/zfs/arc.c

```diff
@@ -7171,6 +7191,10 @@ arc_tuning_update(void)
 	if (zfs_arc_min_prefetch_lifespan)
 		arc_min_prefetch_lifespan = zfs_arc_min_prefetch_lifespan;
 
+	/* Valid range: 1 - N ticks */
+	if (zfs_arc_min_long_lifespan)
```

Comment:
I didn't want to do this to match the style of `zfs_arc_min_prefetch_lifespan` a few lines above it